docs/operations.md

```diff
@@ -18,10 +18,52 @@ then deleting the old chunks. The scavenged events are also removed from the ind
 
 Running a scavenge causes the active chunk to be completed so that it can be scavenged.
 
-::: warning 
+::: warning
 Scavenging is destructive. Once a scavenge has run, you cannot recover any deleted events except from a backup.
 :::
 
+### Mechanism of the scavenging
+
+Running scavenge for the first time will take longer compared to subsequent scavenges because it needs to create a "scavenge point" as part of the first phase, known as the accumulation phase. This is followed by two more phases at the end of which events are removed from the database, including from the current chunk. More information on these phases can be found below.
+
+Whenever a scavenging process starts on a node, it checks if there are any existing unused scavenge point on that. If there are, it utilizes one of them; otherwise, it creates a new scavenge point and replicates it to another node.
```

Comment:
- Can you we also mention that every time scavenging starts, the current active chunk is closed? 
- Can we also mention that the scavenge point is replicated to other nodes. So you can run scavenging up to the same scavenging point on another nodes using the SyncOnly parameter.